Region/Region.cc

```diff
@@ -490,9 +480,11 @@ bool Region::MakeExtensionBox(casacore::WCBox& extend_box, int stokes, ChannelRa
 }
 
 casacore::WCRegion* Region::MakeExtendedRegion(int stokes, ChannelRange channel_range) {
```

Comment:
could these not return a `shared_ptr`?

Reply:
I was initially going to convert all of these to return `shared_ptr` -- but the casacore functions return C pointers anyway, so there has to be a conversion at some point anyway. I was worried about knock-on effects. But I can revisit this. Is there a specific place where we would benefit from using `shared_ptr`s here?